library/junos_install_config

```diff
@@ -250,11 +261,21 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
-    logging.info("connecting to host: {0}@{1}:{2}".format(args['user'], args['host'], args['port']))
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
```

Comment:
The idea here is to match the exact same things that Ansible would normally interpret as a boolean.

From http://docs.ansible.com/ansible/YAMLSyntax.html:
>Ansible doesn’t really use these too much, but you can also specify a boolean value (true/false) in several forms:

create_key: yes
needs_agent: no
knows_oop: True
likes_emacs: TRUE
uses_cvs: false

The current code would not match the string `TRUE`, for example.
The Ansible code for determining boolean values does a case-insensitive match on the boolean values. You can see this here:
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L2046-L2057

That's why I'm suggesting this line should be:
```
if str(args['ignore_warning']).lower() in BOOLEANS + "none":
```
That should match the same values which `module.boolean()` will actually convert into a boolean value.